core/common/src/main/java/alluxio/metrics/MetricKey.java

```diff
@@ -773,6 +773,28 @@ public MetricKey build() {
           .setMetricType(MetricType.COUNTER)
           .setIsClusterAggregated(true)
           .build();
+  // Client local cache metrics
+  public static final MetricKey CLIENT_CACHE_BYTES_READ_CACHE =
+      new Builder(Name.CLIENT_CACHE_BYTES_READ_CACHE)
+          .setDescription("Total number of bytes read from the Local Cache.")
```

Comment:
nit: should `Local Cache` be capitalized?

Reply:
Updated: #10907